src/components/markdown-preview/markdown-preview.jsx

```
@@ -0,0 +1,117 @@
+    if (inOrderedList) {
+        processedLines.push('</ol>');
+    }
+    
+    return processedLines.join('\n');
+};
+
+const MarkdownPreview = ({className, content}) => (
+    <div className={classNames(styles.markdownPreview, className)}>
+        <div 
+            className={styles.content}
+            dangerouslySetInnerHTML={{__html: parseMarkdown(content)}}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)